es-base-code/main/default/objects/Customer_Fields__mdt/fields/Customer_City__c.field-meta.xml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>Customer_City__c</fullName>
-    <description
-    >Used because Address compound fields aren't yet available as independent fields in entity relationships.</description>
     <externalId>false</externalId>
     <fieldManageability>SubscriberControlled</fieldManageability>
     <inlineHelpText
-    >Enter the API name of the field that holds the customer's city information.</inlineHelpText>
+    >Enter the API name of the field that holds the customer&apos;s city information.</inlineHelpText>
```

Comment:
I think we can remove the help text. None of the other fields which have the Field Def lookup have help texts.

Reply:
done!